xspf2m3u.py

```diff
@@ -21,7 +21,8 @@
 EXTS = ['mp3', 'ogg', 'opus', 'mp4', 'm4a', 'wav', 'flac', 'wma']
 CHARS = re.compile('[^a-z0-9 ]')
 
-__version__ = '0.0.0'
+#Bump version 0.1.0 with example and M3UEXT support
+__version__ = '0.1.0'
```

Comment:
Please don't change the version number. I will do that when I make the release.

Reply:
ok